### DISCUSSION TITLE
Add border to arrow?

### DISCUSSION THREAD

**Discussion by LuqmanAzman** [gilbarbara/react-joyride#1009] [Q&A] [answered]
Hi, how can I add border to arrow? I only able to change the color but cannot add border to arrow. Here's how I use floaterProps

```
floaterProps={{
          styles: {
            arrow: {
              // color: 'purple',
              border: '5px solid #face09 !important',
            },
          },
        }}
```

**Comment by gilbarbara** [comment, marked as answer]
Hey @LuqmanAzman 
It's not supported. Adding a border to the arrow alone produces weird results:

![floater-arrow-border](https://github.com/gilbarbara/react-joyride/assets/31954/9bdd0650-0ca0-44b1-b913-7152603a6eae)

If you still want to add it, you could do it with CSS.
```css
.__floater__arrow {
	polygon {
		stroke: #f04;
    	stroke-width: 3px;
	}
}
```
